runbooks/fault-proofs.md

```diff
@@ -0,0 +1,356 @@
+# Key Handover Runbook
+
+This document describes how to generate upgrade playbooks to upgrade chains from `op-contracts/v1.3.0` (MCP) to `op-contracts/v1.X.0` (Fault Proofs).
+
+## Upgrades Involved
+
+This runbook will apply the following upgrade:
+
+- [Fault Proofs](#fault-proofs-upgrade-7)
+
+Each upgrade is described below.
+The Extended Pause upgrade is only applied to chains currently on the Bedrock commit, corresponding to the [`op-contracts/v1.0.0`](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.0.0) tag in the [Optimism monorepo](https://github.com/ethereum-optimism/optimism).
```

Comment:
Extended pause is 1.2.0: https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts/v1.2.0